### DISCUSSION TITLE
404: Resource not found

### DISCUSSION THREAD

**Discussion by Kongstad** [microsoft/PlanetaryComputer#351] [General]
Hi,

I am using MSPC quite alot and trying to spread the word to my colleagues. However, since you announced the retirement of the Hub, it is no longer possible to request an account here: https://planetarycomputer.microsoft.com/account/request
I assume an API key is still required to access the Sentinel-1 RTC data. So how does new users go about doing that these days?

I really hope you are not planning on stopping the general service! It is extremely useful!

Best regards

**Comment by mmcfarland** [comment]
Hi @kongstad, thanks for your interest in the Planetary Computer. We're no longer requiring API key access for Sentinel-1 RTC, or any dataset, in our catalog. Anonymously requested SAS tokens will be able to access the RTC data, and so accounts requests are no longer needed. 

Ref: https://planetarycomputer.microsoft.com/docs/concepts/sas/

**Reply by Kongstad** [reply]
Thank you for your reply
